client/public/index.html

```diff
@@ -1,4 +1,297 @@
 <!DOCTYPE html>
+<html lang="en">
```

Comment:
Typically in a React application, it's not recommended to add code directly to the `index.html` file. This [source ](https://www.freecodecamp.org/news/quick-guide-to-understanding-and-creating-reactjs-apps-8457ee8f7123/)could be helpful to find out the reason for that.